Mail/Views/Thread/Message/Attachment/AttachmentsView.swift

```diff
@@ -43,51 +43,74 @@ struct AttachmentsView: View {
     }
 
     var body: some View {
-        VStack(spacing: UIPadding.regular) {
+        VStack(alignment: .leading, spacing: UIPadding.regular) {
             ScrollView(.horizontal, showsIndicators: false) {
                 HStack(spacing: UIPadding.small) {
                     ForEach(attachments) { attachment in
                         Button {
                             openAttachment(attachment)
                         } label: {
-                            AttachmentCell(attachment: attachment)
+                            AttachmentView(title: attachment.name, subtitle: attachment.size.formatted(.defaultByteCount), icon: attachment.icon)
+                        }
+                    }
+                    if let swissTransferAttachment = message.swissTransferAttachment {
+                        ForEach(swissTransferAttachment.files) { file in
+                            Button {
+                                downloadSwissTransferAttachment(stUuid: swissTransferAttachment.uuid, fileUuid: file.uuid)
+                            } label: {
+                                AttachmentView(title: file.name, subtitle: file.size.formatted(.defaultByteCount), icon: file.icon)
```

Comment:
Maybe we can have a AttachmentView(attachment: Attachment...) and a AttachmentView(swissTransferFile: ...) to simplify the code